deluge/ui/util/lang.py

```diff
@@ -0,0 +1,114 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2007,2008 Andrew Resch <andrewresch@gmail.com>
+#
+# This file is part of Deluge and is licensed under GNU General Public License 3.0, or later, with
+# the additional special exception to link portions of this program with the OpenSSL library.
+# See LICENSE for more details.
+#
+
+import gettext
+import locale
+import logging
+import os
+import sys
+
+import deluge.common
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())  # Silence: No handlers could be found for logger "deluge.util.lang"
+
+
+def set_dummy_trans():
+    import __builtin__
+    __builtin__.__dict__["_"] = lambda x: x
```

Comment:
should add a log warning that translation is unavailable but strings have been marked for translation